pkg/helm/config.go

```diff
@@ -330,9 +330,19 @@ func valueWithDefault(name string, def int) (int, error) {
 }
 
 func getImageNameTag(envValue string) (string, string) {
-	img := strings.Split(envValue, ":")
+	var img []string
+	var repoPath string
+
+	pos := strings.LastIndex(envValue, "/")
+	if pos > 0 {
+		img = strings.Split(envValue[pos+1:], ":")
+		repoPath = envValue[:pos] + "/"
+	} else {
+		img = strings.Split(envValue, ":")
+	}
```

Comment:
we usually refrain from using elses, here I think you can just change this to:
```go
	pos := strings.LastIndex(envValue, "/")
	img := strings.Split(envValue[pos+1:], ":")
	repoPath := envValue[:pos+1]
```

Reply:
fixed